crates/trycp_server/src/app_interface.rs

```diff
@@ -44,12 +48,27 @@ pub(crate) async fn connect(
         return Ok(());
     }
 
-    let mut url = url::Url::parse("ws://localhost").expect("localhost to be valid URL");
-    url.set_port(Some(port)).expect("can set port on localhost");
-
-    let (ws_stream, _resp) = tokio_tungstenite::connect_async(url)
+    let addr = format!("127.0.0.1:{port}");
```

Comment:
I really dislike that we have to do this, I've raised an issue for it on Holochain https://github.com/holochain/holochain/issues/3571